dali/operators/numba_function/numba_func.cc

```diff
@@ -131,7 +134,7 @@ DALI_SCHEMA(NumbaFuncImpl)
   .AddOptionalArg<int>("setup_fn", R"code(Address of setup function setting shapes for outputs.
 This function is invoked once per batch.)code", 0)
   .AddOptionalArg("batch_processing", R"code(Determines whether the function is invoked once per batch or
-separately for every sample in the batch.)code", false);
+separately for each sample in the batch.)code", false);
```

Comment:
Shouldn't the explanation be here, too?

Reply:
Perhaps it should be here only.

Reply:
This isn't shown in our documentation, it has `MakeInternal()` option.